v21.1/add-region.md

```diff
@@ -46,7 +46,12 @@ ALTER DATABASE foo PRIMARY REGION "us-east1";
 ALTER DATABASE PRIMARY REGION
 ~~~
 
-For more information, see [Database regions](multiregion-overview.html#database-regions).
+Given a cluster with multiple regions, any databases in that cluster that have not yet had their primary regions set will have their replicas spread as broadly as possible for resiliency. When a primary region is added to one of these databases:
+
+- All tables will be [`REGIONAL BY TABLE`](set-locality.html#regional-by-table) by default.
```

Comment:
We might want to say here "REGIONAL BY TABLE IN PRIMARY REGION" or REGIONAL BY TABLE in the primary region.

Reply:
Updated - thanks for the review!